ExpenseTracker/ExpenseTracker.Infrastructure/Email/EmailService.cs

```diff
@@ -1,25 +1,19 @@
-﻿using ExpenseTracker.Infrastructure.Email.Interfaces;
+﻿using ExpenseTracker.Infrastructure.Configurations;
+using ExpenseTracker.Infrastructure.Email.Interfaces;
 using MailKit.Net.Smtp;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
 using MimeKit;
 
 namespace ExpenseTracker.Infrastructure.Email;
 
 public class EmailService : IEmailService
 {
-    private readonly string _from;
-    private readonly string _smtpServer;
-    private readonly int _port;
-    private readonly string _username;
-    private readonly string _password;
+    private readonly EmailOptions _options;
 
-    public EmailService(IConfiguration configuration)
+    public EmailService(IOptionsMonitor<EmailOptions> options)
```

Comment:
are we going to change email configurations dynamically? If not we can simply use **IOptions<T>**